examples/plot_wlasso.py

```diff
@@ -89,7 +89,7 @@
 criterion = HeldOutMSE(idx_train, idx_val)
 algo = ImplicitForward()
 monitor = Monitor()
-optimizer = LineSearch(n_outer=20, tol=1e-6)
+optimizer = LineSearch(n_outer=20, tol=1e-6, verbose=True)
 grad_search(algo, criterion, model, optimizer, X, y, log_alpha0, monitor)
```

Comment:
while you're at it, L93 results in a large block of text being displayed on the example (on the doc website), can you assign the result to a variable?